### ISSUE TITLE
New option to "include your memories as a downloadable file" 

### ISSUE THREAD

**Issue by musoobi** [ToTheMax/Snapchat-All-Memories-Downloader#74]
There's a new toggle at https://accounts.snapchat.com/accounts/downloadmydata that reads: Do you want to include your memories as a downloadable file?

Waiting for the export to see what we get!

<img width="394" alt="download" src="https://user-images.githubusercontent.com/116320965/230823612-8c207122-2eee-4a00-9cfb-f12d9c42c9f3.png">

**Comment by ToTheMax** [comment]
Yes I noticed! The only thing is that Snapchat doesn't add dates with this method. When you disable the option and do it with my script it will fix the dates (I updated the README)
